src/zc/buildout/buildout.py

```diff
@@ -2161,8 +2161,6 @@ def main(args=None):
                     _help()
                 elif orig_op == '--version':
                     _version()
-                elif orig_op == '--restart-after-upgrade':
-                    options.append(('buildout', 'restart-after-upgrade', "true"))
```

Comment:
I'm curious as to what happens if a zc.buildout before this patch upgrades itself to a version after this patch.  Will the old buildout add --restart-after-upgrade to the command line, causing the new buildout to fail with an unknown command line argument error?

Reply:
@mgedmin  ``--restart-after-upgrade`` only exist in 3.0.0a1 and 3.0.0a2

I feel we do not need to cover those alpha versions. What is your take on that ?

Reply:
Good point!  I forgot when it was introduced; seemed like ages ago.